tensorflow/tensorboard/components/tf-dashboard-common/dashboard-style.html

```diff
@@ -54,6 +54,14 @@
         display: block;
       }
 
+      .log-option-button {
```

Comment:
I think keeping it on dashboard-style for now is ok, I'll have to move a bunch of css rules in the future anyway.